internal/compiler/widgets/common/listview.slint

```diff
@@ -32,6 +33,7 @@ component StandardListViewBase inherits ListView {
     private property <length> into-view-item-y: root.item-y(root.into-view-item);
     private property <length> current-item-y: root.item-y(root.focus-item);
     private property <int> focus-item: 0;
+    private property <bool> first-selection: false;
```

Comment:
Not very happy with this name... If someone can come up with something better I'll gladly update it.

Reply:
Not sure what this is meant to do so i can't think of a better name. 
But I notice that it is never reset to false. Should it perhaps be rest when the view gain focus or something like that?